scheduler/src/cook/mesos/data_locality.clj

```diff
@@ -1,32 +1,118 @@
 (ns cook.mesos.data-locality
-  (:require [cook.config :as config]
-            [plumbing.core :as pc])
+  (:require [cheshire.core :as cheshire]
+            [clj-http.client :as http]
+            [clj-time.core :as t]
+            [clojure.set :as set]
+            [cook.config :as config]
+            [cook.mesos.util :as util]
+            [plumbing.core :as pc]
+            [clojure.tools.logging :as log]
+            [datomic.api :as d])
   (:import com.netflix.fenzo.VMTaskFitnessCalculator
-           com.netflix.fenzo.plugins.BinPackingFitnessCalculators))
+           com.netflix.fenzo.plugins.BinPackingFitnessCalculators
+           java.util.UUID))
+
+(let [data-local-costs (atom {})
```

Comment:
Should we expose these data on some admin endpoint for easing debugging data locality support?

Reply:
Done